src/main/java/com/fp3/haras/view/screens/auth/LoginScreen.java

```diff
@@ -21,6 +25,16 @@ public class LoginScreen extends javax.swing.JFrame {
     }
     
     public LoginScreen() {
+        try {
+            User user = EntityUtils.select("SELECT u FROM User u WHERE u.id = 1", User.class).get(0);
+            this.user = user;
+        } catch (Exception e) {
+            if (user == null) {
+                this.user = new User("admin", "admin");
+                EntityUtils.insert(this.user);
+            }
+        }
```

Comment:
This is done because if the user is running the program for the first time on their machine, there's still no users on the table, so we create a default one (admin admin)